content/zh-cn/docs/concepts/overview/_index.md

```diff
@@ -257,7 +257,7 @@ Kubernetes：
 * Does not provide nor adopt any comprehensive machine configuration, maintenance, management, or self-healing systems.
 * Additionally, Kubernetes is not a mere orchestration system. In fact, it eliminates the need for orchestration. The technical definition of orchestration is execution of a defined workflow: first do A, then B, then C. In contrast, Kubernetes comprises a set of independent, composable control processes that continuously drive the current state towards the provided desired state. It shouldn't matter how you get from A to C. Centralized control is also not required. This results in a system that is easier to use and more powerful, robust, resilient, and extensible.
 -->
-* 不是日志记录、监视或警报的解决方案。
+* 不提供日志记录、监视或警报的解决方案。
```

Comment:
* Does not dictate logging, monitoring, or alerting solutions. It provides some integrations as proof of concept, and mechanisms to collect and export metrics.

The above is the original text, I didn't see the description about the offer.
I don't think this change is necessary.

Reply:
You are right, I forgot to read the original text.

Reply:
Thanks.